test/spec/Feature/Query/SingularSpec.hs

```diff
@@ -85,7 +85,7 @@ spec =
             [("Prefer", "tx=commit"), ("Prefer", "return=representation"), singular]
             [json| { address: "zzz" } |]
           `shouldRespondWith`
-            [json|{"details":"Results contain 4 rows, application/vnd.pgrst.object+json requires 1 row","message":"JSON object requested, multiple (or no) rows returned","code":"PGRST116","hint":null}|]
+            [json|{"details":"Results contain 4 rows, application/vnd.pgrst.object+json requires 1 row","message":"JSON object requested, multiple (or no) rows returned","code":"PGRST116","hint":"4"}|]
```

Comment:
@taimoorzaeem Besides the hint (which is likely unnecessary now), we can make an improvement to the error message:

```suggestion
            [json|{"details":"The result contains 4 rows","message":"JSON object requested, multiple (or no) rows returned","code":"PGRST116"}|]
```

The reasoning is that I've been noticing that users ignore the error message when they see the `application/vnd.pgrst.object+json`. This happens when they use PostgREST through client libraries, like https://github.com/supabase/postgrest-js.

Reply:
Also technically the error message is not accurate since it should refer to json array elements or array length and not "rows". It could be reworded.

Reply:
@steve-chavez "rows" is more consistent with other error messages and also since "rows" refer to the rows returned from the database so it is accurate IMO. Perhaps maybe no need to reword that.